openff/toolkit/topology/topology.py

```diff
@@ -2402,24 +2402,16 @@ def is_constrained(self, iatom, jatom):
             return False
 
     @requires_package("nglview")
-    def visualize(self, ensure_correct_connectivity: bool = False) -> "NGLWidget":
+    def visualize(self) -> "NGLWidget":
         """
         Visualize with NGLView.
 
-        Requires all molecules in this topology have positions.
+        Requires that all molecules in this topology have positions.
 
         NGLView is a 3D molecular visualization library for use in Jupyter
-        notebooks. Note that for performance reasons, by default the
-        visualized connectivity is inferred from positions and may not reflect
-        the connectivity in the ``Topology``.
-
-        Parameters
-        ==========
-
-        ensure_correct_connectivity: bool, default=False
-            If ``True``, the visualization will be guaranteed to reflect the
-            connectivity in the ``Topology``. Note that this will severely
-            degrade performance, especially for topologies with many atoms.
+        notebooks. Note that the visualized molecule may, in strained
+        conformations, include bonds not present in the topology. Rendering
+        multiple bonds additionally requires RDKit.
```

Comment:
(not blocking) Is this the correct interpretation, or am I misunderstanding the intent here?

```suggestion
        conformations, include bonds not present in the topology. Rendering
        bond orders correctly additionally requires RDKit.
```